docs/learn/clientside-interpolation.md

```diff
@@ -1,42 +1,29 @@
+This is a conservative approach that never shows an incorrect user state, but exposes users directly to the effects of latency. Regardless of the client's potential frame rate, the game will only ever run at the speed the server (with its limiting [network factors](lagandpacketloss.md#network-latency)) is capable of, which can severely reduce frame rates and produce perceivable input lag. It also produces an uneven game speed, as jitter in the network causes states to arrive and be rendered at inconsistent intervals.
 
-Now throw in some chaos by means of [RTT fluctuations](lagandpacketloss.md#round-trip-time-rtt) and [Jitter](lagandpacketloss.md#jitter), and we are getting into an unacceptable gameplay experience.
+In addition to reduced responsiveness, this approach can also directly interfere with gameplay in certain genres such as first-person shooters: because the information from the server is always slightly behind, due to network latency, players are forced to aim ahead of their target to compensate.
 
-Fortunately we can mitigate the effects of latency and jitter.
+## With client-side interpolation
 
-## Client-Side Interpolation
+When using client-side interpolation, all clients intentionally run slightly behind the server, giving them time to transition smoothly between state updates and conceal the effects of latency from users. Client-side interpolation is implemented in the [NetworkTransform](../components/networktransform.md) component.
 
-Visual choppiness is caused by infrequent (in comparison to the speed at which clients are rendering their screens) updates from the server and it can also be exasperated by bad network conditions causing jitter. We can employ **Client-Side Interpolation** to reduce this effect.
+In a standard client-server [topology](../terms-concepts/network-topologies.md), clients are able to render a state that's approximately half the [round-trip time (RTT)](lagandpacketloss.md#round-trip-time-and-pings) behind the server. When using client-side interpolation, a further intentional delay is added so that by the time the client is rendering state _n_, it's already received state _n+1_, which allows the client to always smoothly transition from one state to another. This is effectively increasing latency, but in a consistent, client-universal way that can be used to reduce the negative impacts of unpredictable network jitter on gameplay.
 
-In client-side Interpolation instead of just snapping objects to their positions that are transmitted from the server the client smoothly interpolates to this state over time. This approach is still conservative - the client just smoothens out the transition between valid states that were sent from the server.
+The delay added to enable client-side interpolation is referred to as the interpolation period, and should always be less than the interval the server waits between sending packets to clients, to avoid stutter.
```

Comment:
Referred to where? Otherwise could say "called"?

Reply:
Agreed, changing.